### PR TITLE
ARM 64 and release/dbg build fixes

### PR DESCRIPTION
**Pull request by devnexen**
Fix script data overflows detected by Asan

### CODE REVIEW

**Comment by devnexen** [comment]
Tested on a Mac M1

File: src/iscreen/iscreen.h [hunk 1]
```diff
@@ -33,7 +33,7 @@ const int  iS_STR_SEG_START_COL	= 144;
 //const int  iS_STR_SEG_START_COL	  = 80;
 
 const int  iSTR_LEN			= 50;
-const int  iS_STR_LEN		= 100;
+const int  iS_STR_LEN		= 512;
```
Comment (stalkerg) [inline src/iscreen/iscreen.h:36]: Why you changed this length?

Comment (devnexen) [reply src/iscreen/iscreen.h:36]: That.s the reason of the overflows, the length of data read were bigger than this.

**Comment by stalkerg** [comment]
Thanks!

**Pull request merged**